Assets/Scripts/Model/Player.cs

```diff
@@ -21,5 +26,33 @@ public void AddToScoreLong(long bonus)
         {
             score += bonus;
         }
+
+
+        private void OnControllerColliderHit(ControllerColliderHit hit)
+        {
+
+            Exit exit = hit.gameObject.GetComponent<Exit>();
```

Comment:
Myślę, że powinniśmy raczej przypisać wyjściu tag "Exit" i sprawdzać czy gameObject, z którym jest kolizja ma tag "Exit" wtedy dopiero wołać GetComponent<Exit>.
Ogólnie sprawdzanie kolizji będzie musiało także działać dla kolizji z potworkami, więc trzeba to jakoś rozróżnić